sort.php

```diff
@@ -49,7 +49,7 @@ function bubbleSort($sort_array, $checkEquality)
 
 $optind = null;
 $options = getopt("n", [], $optind);
-if ($options) {
+if (! empty($options)) {
```

Comment:
You're just checking for whether the user has supplied _any_ options. Yes, there's only one at the moment, but have a look at the issue list...